### DISCUSSION TITLE
Ability to select all files in the current view.

### DISCUSSION THREAD

**Discussion by zaaakher** [spacedriveapp/spacedrive#2419] [Feature Request]
I naturally did `ctrl + a` to select all the file but it seems that doesn't work. I looked for it in the settings > keybinds but I couldn't find it.

So I think maybe we're missing the ability to select all files/folders with the good ol' `ctrl + a`

**Comment by utkubakir** [comment]
https://github.com/spacedriveapp/spacedrive/issues/2421 weird I thought we had this haha

**Comment by ketonik** [comment]
haha yeah this was spinning me out too, lol...my keyboard not working eh eh
